api/src/wfl/api/workloads.clj

```diff
@@ -59,6 +59,16 @@
                 :type  ::workload-not-found})))
     (try-load-workload-impl tx (first workloads))))
 
+(defn load-workload-for-project
```

Comment:
I think it's possible to have more than one workload for a  given project. In which case, I'd consider returning a list of them - perhaps rename `load-workloads-for-project` and do something similar to `load-workloads`?

Reply:
Ah, that's a great point, somehow I pre-assumed the project and workload is 1-1 mapping which indeed isn't. Will update!